src/Workspaces/Core/Portable/PublicAPI.Unshipped.txt

```diff
@@ -1,3 +1,4 @@
+*REMOVED*abstract Microsoft.CodeAnalysis.Editing.SyntaxGenerator.ParameterDeclaration(string name, Microsoft.CodeAnalysis.SyntaxNode type = null, Microsoft.CodeAnalysis.SyntaxNode initializer = null, Microsoft.CodeAnalysis.RefKind refKind = Microsoft.CodeAnalysis.RefKind.None) -> Microsoft.CodeAnalysis.SyntaxNode
```

Comment:
this is the replacement of an abstract method with a non-virtual one.  This is always safe as this is a type that has an internal cosntructor (e.g. it's not publicly subclassable).